docs/FRSKY_R9.md

```diff
@@ -184,47 +184,47 @@ Button:
 
 ## Flashing ##
 
-The mLRS firmware must, of course, be flashed on both the Tx module and the receiver before it can be used. For flashing, as well as upgrading, several methods are available (via ExpressLRS bootloader, via system bootloader, via STLink/SWD), which are described in the following. 
+The mLRS firmware must, of course, be flashed on both the Tx module and the receiver before they can be used. For flashing, as well as upgrading, several methods are available: via ExpressLRS bootloader, via ST-Link/SWD, or via system bootloader. The methods are each described in the following. 
```

Comment:
Small rewrite of first sentence:

'The mLRS firmware must be flashed on both the Tx module and the receiver before they can be used.'

Perhaps bullets for the three methods?

Reply:
done
I think the 'bullets' are coming as titles :)